src/App.tsx

```diff
@@ -1,8 +1,66 @@
-import React from 'react';
+import React, { FC, useState } from 'react';
+import { getTodos, getUsers } from './Api';
+import { TodoList } from './components/TodoList/TodoList';
+
 import './App.css';
 
-const App = () => (
-  <h1>Dynamic list of TODOs</h1>
-);
+const App: FC<{}> = () => {
+  const [todos, setTodos] = useState<TodosWithUser>([]);
+  const [isLoading, setLoading] = useState<boolean>(false);
+  const [isStarted, setStarted] = useState<boolean>(true);
```

Comment:
примитивы необязательно указывать, они устанавливаются автоматически
```suggestion
  const [isLoading, setLoading] = useState(false);
  const [isStarted, setStarted] = useState(true);
```